app/views/layouts/tylium/_nodes.html.erb

```diff
@@ -1,5 +1,5 @@
 <ul class="nodes-nav tree-navigation">
-  <li class="tree-header sidebar-link m-0 <%= 'active' if @node %>" data-behavior="sidebar-toggle open-only">
+  <li class="tree-header sidebar-link m-0 <%= 'active' if @node && !@board %>" data-behavior="nodes-tree sidebar-toggle open-only">
```

Comment:
added the check for `!@board` because boards#show was triggering an active state for both nodes and methodologies in the sidebar (at the same time)

Reply:
But this is a check on what "is not", can you replace it with a check on what "is"? For instance, something about the nodes_scope or the /nodes/ path or controller name?

Reply:
fixed